src/source/view/controls/ValidationUtils.bs

```diff
@@ -0,0 +1,49 @@
+import "pkg:/source/core/BaseClass.bs"
+
+class InputValidator extends mc.BaseClass
```

Comment:
ferw things here:
1. can we get a unit test - ping me, and we'll add it together, and I'll show you how we test things.
2. your indtenation is wacky - please ensure you have format-on-save enabled 
3. always cameCalse, please :)